x/oracle/types/keys.go

```diff
@@ -33,7 +34,7 @@ var (
 // KeyExchangeRate - stored by *denom*
 func KeyExchangeRate(denom string) []byte {
 	// append 0 for null-termination
-	return util.ConcatBytes(1, KeyPrefixExchangeRate, []byte(denom))
+	return util.ConcatBytes(1, KeyPrefixExchangeRate, []byte(strings.ToUpper(denom)))
```

Comment:
this key will override previous stored price of denom

we need to store the prices with timestamps for storing previous prices
```
tbz, _  := timestamp.MarshalBinary()
key = util.ConcatBytes(1, KeyPrefixExchangeRate, []byte(strings.ToUpper(denom)),tbz)
```

Reply:
Ok